src/lora_region.erl

```diff
@@ -0,0 +1,1627 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%% Copyright (c) 2016-2019 Petr &lt;Gotthard petr.gotthard@@centrum.cz&gt;
+%% All rights reserved.
+%% Distributed under the terms of the MIT License. See the LICENSE file.
+%% @end
```

Comment:
lora_region is an **exact** copy of the original router Region code.  It has several purposes but it should never be used outside this LoRaWAN library.  It is used for eunit testing only.
1. The lora_region functions are used to eunit test the new lora_plan functions.
2. It served to bootstrap the new lora_plan code because writing good test code is hard.
3. It is a useful sanity check on the new lora_plan functionality.

Reply:
lora_region is now renamed test_region.
Also test_region is now protected with EUNIIT ifdef